includes/abstracts/abstract-wp-job-manager-email-template.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Abstract class for an email notification built using templates.
+ *
+ * @package wp-job-manager
+ *
+ * @since 1.31.0
+ */
+
+abstract class WP_Job_Manager_Email_Template extends WP_Job_Manager_Email {
+
+	/**
+	 * Get the rich text version of the email content.
+	 *
+	 * @return string
+	 */
+	public function get_rich_content() {
+		return $this->get_template( false );
+	}
+
+	/**
+	 * Get the plaintext version of the email content.
+	 *
+	 * @return string
+	 */
+	public function get_plain_content() {
+		if ( $this->has_template( true ) ) {
+			return $this->get_template( true );
```

Comment:
`get_template` may return a bool, whereas this function claims to return a string. Is this ok?

Reply:
I'll change the signature of `get_template()` method 5e9d1a0f8297d9275c7c0703c613e7085fa8f817